ml-agents/mlagents/trainers/tests/test_settings.py

```diff
@@ -31,6 +32,32 @@ def check_if_different(testobj1: object, testobj2: object) -> None:
                 check_if_different(val, attr.asdict(testobj2, recurse=False)[key])
 
 
+def check_dict_is_at_least(
```

Comment:
This addition will conflict with the same change in #4174. Not a blocker, just want to make sure this is intended.

Reply:
I think once the curriculum refactor is done, this version of the test will work fine for that PR as well. I removed the exclusion for `environment_parameters` since it's not needed on `master`.